v0.25: Task api (#1208)
meilisearch/documentation: reference/api/README.md

**201 - Created**: The resource has been created (synchronous)

> **202 - Accepted**: The task has been pushed in the task queue (asynchronous)

Reviewer: ```suggestion
**202 - Accepted**: The task has been added to the queue (asynchronous)
```
